pyproject.toml

```diff
@@ -35,7 +35,7 @@ known_third_party = ["click","google","kedro","kfp","kubernetes","tabulate", "py
 python = ">=3.8,<3.11" # weird error OverrideNeeded when bumping up. Could be related to old versions with kfp
 kedro = ">=0.19.0,<0.20"
 click = ">=8.0.4"
-kfp = ">=1.8.21,<2.0"
+kfp = "2.8.0"
```

Comment:
can this requirement be softened? It's not okay for secondary libraries to make very strict version restrictions

Reply:
Good point, thank you! I'll soften this requirement, possibly just `kfp > "2.0"` wdyt?

Reply:
Yes, I think it's fine, >= 2.0. Let's maybe also test for 2.0 version to be make sure nothing breaks between those versions.

Reply:
Ok, I'll do that!

Reply:
`>=2.0, <3`